nistats/first_level_model.py

```diff
@@ -583,6 +585,94 @@ def compute_contrast(self, contrast_def, stat_type=None,
 
         return outputs if output_type == 'all' else output
 
+    def get_voxelwise_model_attribute_(self, attribute, result_as_time_series=True):
+        """Transform RegressionResults instances within a dictionary
+        (whose keys represent the autoregressive coefficient under the 'ar1'
+        noise model or only 0.0 under 'ols' noise_model and values are the
+        RegressionResults instances) into input nifti space.
+
+        Parameters
+        ----------
+        attribute : str
```

Comment:
attribute is defined within a fixed dictionary of values I guess ? This should be provided if the function is meant to be used publicly.

Reply:
I believe this should really be a private method, but I added additional documentation and an internal check to be explicit anyway.

Reply:
Names of private methods start with an underscore. We should change `get_voxelwise_model_attribute_` to `_get_voxelwise_model_attribute_`. 

What's the underscore at the end for?

Reply:
Nm, oudated comment.